public/main.js

```diff
@@ -538,24 +538,18 @@ const mapComp = Vue.component('map-comp', {
     data() {
         return {
             tooltip: null,
+            mapData: {},
+            loaded: false,
             // input: "0,0",
-            // debug: "",
-            // debug: "hide-input",
         }
     },
     mounted() {
-        if (this.$refs.tooltip) {
-            this.tooltip = Popper.createPopper(this.$refs.slot, this.$refs.tooltip, {
-                placement: 'right',
-                modifiers: [{ name: 'offset', options: { offset: [0, 5] } }],
+        fetch("https://map.runfast.stream/js/tilesources.json")
+            .then(res => res.json())
+            .then(data => {
+                this.mapData = data
+                this.loaded = true
             })
```

Comment:
Do something on failure, too with `.catch(err => ....)`, even if it's just displaying some text like "failed to fetch map data"